### DISCUSSION TITLE
News menus not outputting data

### DISCUSSION THREAD

**Discussion by rica-carv** [e107inc/e107#5294] [Q&A Plugins]
There's three menus from the news core plugin that are not rendering any output: `news_month_menu`, `other_news_menu` & `other_news2_menu`

Is this normal?

**Comment by Vodhin** [comment]
I'm not sure about the `news_month_menu`, but have you checked to see if you have articles set to these locations? (in the Advanced tab).

![othernews](https://github.com/e107inc/e107/assets/135135923/733dec88-4407-431e-ba96-05e0196826df)

I think there are some preferences settings to modify, too:
![othernews2](https://github.com/e107inc/e107/assets/135135923/ae0f399c-c690-4ade-bc08-59054c9aea98)
